content/lessons/chapter-2/mining-1.tsx

```diff
@@ -188,45 +190,43 @@ export default function Mining1({ lang }) {
     hydrated && (
       <div className="grid grid-cols-1 justify-center justify-items-center md:my-auto md:flex md:flex-row">
         <div className="fade-in grid w-full grid-cols-1 items-center px-[15px] py-[25px] md:order-last md:my-0 md:mx-[30px] md:w-[405px] md:p-[25px]">
-          <div
-            className={clsx(
-              'relative mb-2.5 font-nunito text-lg font-semibold',
-              {
-                'fade-in text-white': blocks !== 0,
-                'text-black/50': blocks === 0,
-              }
-            )}
-          >
-            <span className={clsx({ 'fade-in text-[#EDA081]': blocks !== 0 })}>
-              {t('chapter_two.mining_one.progress_bar_title')}
-            </span>{' '}
-            <span className="absolute right-0">
-              {Intl.NumberFormat().format(blocks)} of 1,000
-            </span>
-          </div>
-          <ProgressBar progress={blocks / 10} />
+          <BlockProgress
+            variant={BlockProgressVariant.TotalBar}
+            fontSize="text-lg"
+            disabled={blocks === 0 && 'text-black'}
```

Comment:
It doesn't make a lot of sense to pass a string to the `disabled` prop, it should be a `boolean`, this is how this prop is commonly being used when creating core components. If you want to apply different css classes based on the disabled state you should do something like this instead:

```tsx
<BlockProgress
  className={clsx({ 'text-black': blocks === 0 })}
  disabled={blocks === 0}
  ...
/>
```

Reply:
Same applies to the other places where you do this.